.github/pull-request-template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please include a summary of the change and which is is fixed.
+
+- Add A
+- Fix B
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix
+- [ ] New feature
+
+## How has this been tested?
+
+Please describe the tests that you ran to verify your changes.
+
+- [ ] Test A
+- [ ] Test B
```

Comment:
```suggestion
[ ] UnitTests
[ ] FunctionalTests
```

Reply:
Using "Test A" would hint to people that they need to describe what exactly was done for the testing

As "UnitTests" can be different in nature
And some PRs should be tested not with tests but rather graphical changes (documentation)
In other cases change might be in the execution speed, so it would be reasonable to ask what was used for checking that (documents under analysis and hardware/software details)  

I would either keep "Test A", or would list few additional options (related to visual check, exec time measurement, and maybe something else)

Reply:
I think keeping `Test A` is good too. There are many variations as @meanrin  told.